src/cli/view/job-status-by-jobid/JobStatusByJobId.Handler.ts

```diff
@@ -20,7 +20,7 @@ export default class ViewJobStatusByJobIdHandler extends FTPBaseHandler {
         params.response.data.setObj(jobDetails);
         params.response.format.output(
             {
-                fields: ["jobName", "jobId", "owner", "status", "rc", "retcode"],
+                fields: ["jobName", "jobId", "owner", "status", "rc", "retcode", "extra"],
```

Comment:
thanks for adding `extra` as part of the default information shown to the user on `view-job-status` 😋